stats/stats/src/charts/lines/active_bundlers.rs

```diff
@@ -0,0 +1,118 @@
+//! Active bundlers on each day.
+
+use std::ops::Range;
+
+use crate::{
+    charts::db_interaction::{read::QueryAllBlockTimestampRange, utils::datetime_range_filter},
+    data_source::{
+        kinds::{
+            local_db::{
+                parameters::update::batching::parameters::Batch30Days, DirectVecLocalDbChartSource,
+            },
+            remote_db::{PullAllWithAndSort, RemoteDatabaseSource, StatementFromRange},
+        },
+        types::BlockscoutMigrations,
+    },
+    ChartProperties, Named,
+};
+
+use blockscout_db::entity::{blocks, user_operations};
+use chrono::{DateTime, NaiveDate, Utc};
+use entity::sea_orm_active_enums::ChartType;
+use migration::{Alias, Expr, Func, IntoColumnRef, IntoIden, SimpleExpr};
+use sea_orm::{
+    ColumnTrait, EntityTrait, IntoIdentity, IntoSimpleExpr, Order, QueryFilter, QueryOrder,
+    QuerySelect, QueryTrait, Statement,
+};
+
+pub(crate) fn count_distinct_in_user_ops(
```

Comment:
i think it's time to create directories for categories in lines. after that move this function to lines/user_ops/mod.rs